app/views/events/_form.html.erb

```diff
@@ -33,6 +33,8 @@
     <div class="col-lg-10">
       <%= f.text_field :max_participants, :class => 'form-control' %>
       <%=f.error_span(:max_participants) %>
+      <%= f.label :participants_are_unlimited %>
```

Comment:
The max_participants value is an integer that has to be greater than 0 for a valid event. As far as I know Ruby Integers (practically) has no limit. So I can't see any (magic) values that can be used to store that information?